opg-php-fpm-1604/docker/my_init.d/40-opg_php_xdebug_remote_host

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
```

Comment:
@tim-mcarthur-moj it seems that the php-fpm package on unbuntu installs the config to the major version number as a directory name by default. Also 16.04 is currently on php7.0.35 by the looks of it. It's been kept consistent with the base php images here